charts/kyverno/values.yaml

```diff
@@ -471,6 +479,15 @@ cleanupJobs:
       seccompProfile:
         type: RuntimeDefault
 
+    resources:
+      # -- Pod resource limits
+      limits:
+        memory: 128Mi
+      # -- Pod resource requests
+      requests:
+        cpu: 100m
+        memory: 64Mi
+
```

Comment:
Do we want default values or leave them empty by default ?

Reply:
You tell me 😉
I thaught adding defaults would be cleaner. The values I added are just a guess.

Reply:
cc @chipzoller @realshuting @treydock what would be the recommendation here ?

Reply:
@bakito please keep default values empty.

Reply:
@eddycharly done